### ISSUE TITLE
Create Stripe integration 

### ISSUE THREAD

**Issue by manuelalferez** [manuelalferez/flowinance#38]
Enhancing the Flowinance model with [Stripe](https://stripe.com/docs/api) integration is a valuable addition.

Tasks:
1. Implement Stripe integration logic.
2. Develop a dedicated payments page.

Let's discuss and collaborate on this! 💬

**Comment by toufiqfarhan0** [comment]
I'would like to work on this

**Comment by manuelalferez** [comment]
@toufiqfarhan0 go ahead! Let me know if I can help with anything ;)

**Comment by toufiqfarhan0** [comment]
![image](https://github.com/user-attachments/assets/e5d27c40-7d01-434f-b379-3430d989ac98)
@manuelalferez you wanna add a card like this??

**Comment by toufiqfarhan0** [comment]
> Yes, cards like that one, where explain what the user gets by paying.

Done!!!

**Comment by manuelalferez** [comment]
Yes, flowinance is connected to supabase. There is where the transactions, logins and users configurations are stored

**Comment by manuelalferez** [comment]
As there has been no activity on this issue for the past 5 days, I will go ahead and unassign it from you

**Comment by toufiqfarhan0** [comment]
> As there has been no activity on this issue for the past 5 days, I will go ahead and unassign it from you

I'm workin on it!! I'm figuring out how where is the schema i need add stripe payment in DB right?

**Comment by manuelalferez** [comment]
Oh, maybe the problems that you are facing are related with: https://github.com/manuelalferez/flowinance/issues/46#issuecomment-2394048457

**Comment by manuelalferez** [comment]
@toufiqfarhan0 please take a moment to read the contributing guidelines: [Contributing Guide](https://github.com/manuelalferez/flowinance/blob/main/CONTRIBUTING.md).

**Comment by toufiqfarhan0** [comment]
> @toufiqfarhan0 please take a moment to read the contributing guidelines: [Contributing Guide](https://github.com/manuelalferez/flowinance/blob/main/CONTRIBUTING.md).

the project running locallly
but I can find DB what DB you using like postgres??

**Comment by manuelalferez** [comment]
@toufiqfarhan0 you should run Supabase locally, Supabase is the database: https://supabase.com/docs/guides/cli/local-development

And there is where you might find problems with the schema

**Comment by toufiqfarhan0** [comment]
@manuelalferez you can unassign the issue so that someone can take up

**Comment by manuelalferez** [comment]
@toufiqfarhan0 okay. What happened?

**Comment by ppranay20** [comment]
@manuelalferez can you tell me more about this issue I would like to work in this.

**Comment by manuelalferez** [comment]
Honestly, since this is an open source project, I see more logical to have a button in the footer with: https://buymeacoffee.com/ and that users who want to support the project do it. Not charge them through Stripe. So I'm going to change the issue description. Thanks for asking! @ppranay20

**Comment by manuelalferez** [comment]
The button should be in the footer, this way it is visible from the whole project.
